app/views/support/PlanOps.scala

```diff
@@ -50,7 +50,7 @@ object PlanOps {
 
     def hasDigitalPack: Boolean = in.products.seq.contains(Digipack)
 
-    def phone: String = "+44 (0) 330 333 6767"
+    def phone: String = "0330 333 6767"
```

Comment:
There are multiple instances of the phone number, throughout a multitude of templates and view classes. We should probably write a story to tidy that up into one helper class with the methods: phoneInternational and phoneUk